server/config.ts

```diff
@@ -34,7 +37,23 @@ export interface ApiConfig {
   agent: AgentConfig
 }
 
+export interface ApplicationInfo {
+  applicationName: string
+  buildNumber: string
+  gitRef: string
+}
+
+const packageJson = path.join(__dirname, '../package.json')
+const { name: applicationName } = JSON.parse(fs.readFileSync(packageJson, { encoding: 'utf8' }))
```

Comment:
`applicationName` could even have been hard-coded right here…

Reply:
in fact, i've hard-coded it now so that the built version doesn't need to look in a different location for package.json file